spec/spec_helper.rb

```diff
@@ -164,6 +164,15 @@
       Msf::FeatureManager.instance.set(Msf::FeatureManager::DATASTORE_FALLBACKS, true)
     end
   end
+
+  # rex-text table performs word wrapping on msfconsole tables:
+  #   https://github.com/rapid7/rex-text/blob/11e59416f7d8cce18b8b8b9893b3277e6ad0bea1/lib/rex/text/wrapped_table.rb#L74
+  # This can cause some integration tests to fail if the tests are run from smaller consoles
+  # This mock will ensure that the tests run without word-wrapping.
+  config.before(:each) do
```

Comment:
I could have added the console size mocking to just the tests that were currently failing, but I imagined the possibility of someone adding brittle tests again in the future - so I just went with globally mocking things to future proof things.